Formula/git-imerge.rb

```diff
@@ -1,7 +1,7 @@
 class GitImerge < Formula
   desc "Incremental merge for git"
   homepage "https://github.com/mhagger/git-imerge"
-  url "https://github.com/mhagger/git-imerge/archive/v1.1.0.tar.gz"
+  url "https://github.com/mhagger/git-imerge/archive/v1.2.0.tar.gz"
   sha256 "62692f43591cc7d861689c60b68c55d7b10c7a201c1026096a7efc771df2ca28"
   license "GPL-2.0"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
```
(guessed from https://github.com/mhagger/git-imerge/blob/0c1c05f631b2b42ae7eded0ea266a29b7871629f/gitimerge.py#L10)

Reply:
Agreed, will modified this